readme.md

```diff
@@ -50,37 +49,65 @@ Adds filetype glyphs (icons) to other plugins such as [NERDTree], [vim-airline],
 <a name="quick-setup"></a>
 ## Quick Setup (TL;DR)
 
-1. Install the plugin per your usual method
-   _[(» More details... «)](#installation)_
-2. Download and install a patched font (or patch your own):
+1. Download and install a patched font (or patch your own):
   * [nerd-fonts]
    _[(» More details... «)](#font-installation)_
-3. Set font  _[(» More details... «)](#font-configuration)_
+2. Set font  _[(» More details... «)](#font-configuration)_
   * a. **vim**: Set your terminal emulator font
   * b. **gvim**: Set `guifont` in your `vimrc`
+3. Install the plugin per your usual method
+   _[(» More details... «)](#installation)_
 
-## Usage
+<a name="installation"></a>
+## Installation
 
-After installing the patched font and setting the vim font just open or look at any of the supported plugins you have installed ([NERDTree][nerdtree], [airline][vim-airline], [powerline], [unite], [lightline.vim], [vimfiler], or [flagship]).
+* [Sample Windows vimrc configuration 1](https://github.com/ryanoasis/vim-devicons/wiki/samples/v0.7.x/.vimrc-windows-1)
+* [Sample Linux vimrc configuration 1](https://github.com/ryanoasis/vim-devicons/wiki/samples/v0.7.x/.vimrc-linux-1)
 
-* _NOTE:_ if you don't have `guifont` set and are not running gvim you will need to set the terminal font.
+This plugin follows the standard runtime path structure, and as such it can be installed with a variety of plugin managers:
 
-* _NOTE:_ for support of these plugins: [NERDTree], [vim-airline], [CtrlP], [powerline], [unite], [vimfiler], [flagship] you **must** configure vim to load those plugins **_before_** vim-devicons loads.
+#### [Pathogen](https://github.com/tpope/vim-pathogen)
+  *  `git clone https://github.com/ryanoasis/vim-devicons ~/.vim/bundle/vim-devicons`
 
-* _NOTE:_ for better [nerdtree-git-plugin] support, you _should_ configure vim to load nerdtree-git-plugin **_before_** vim-devicons loads.
+#### [NeoBundle](https://github.com/Shougo/neobundle.vim)
+  * Add to vimrc:
 
-## Font Configuration
+      ```vim
+      NeoBundle 'ryanoasis/vim-devicons'
+      ```
+  * And install it:
+
+      ```vim
+      :so ~/.vimrc
+      :NeoBundleInstall
+      ```
 
-* Encoding **must** be set to UTF-8 for the glyphs to show
+#### [Vundle](https://github.com/gmarik/vundle)
+  * Add to vimrc:
+
+       ```vim
+       Plugin 'ryanoasis/vim-devicons'
+       ```
+  * And install it:
+
+       ```vim
+       :so ~/.vimrc
+       :PluginInstall`
+       ```
+
+#### Manual
+  *  copy all of the files into your `~/.vim` directory
+
+##### Encoding **must** be set to UTF-8 for the glyphs to show
```

Comment:
Above here can we make another level-4 subheading, something like 'Set Font' or something to separate 'Manual' from the remaining details. Because at a glance it looks like all of this is _under_ just the manual setup instructions :sweat_smile: